### PR TITLE
SD to HD with modifying the luma coefficients

### PR DESCRIPTION
**Pull request by retokromer**
I guess, this answers https://github.com/amiaopensource/ffmprovisr/issues/46

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,7 +205,36 @@
...
 
-  <!-- 16:9 to 4:3 -->
+  <!-- SD to HD -->
+  <span data-toggle="modal" data-target="#SD_HD_2"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Transform SD to HD with pillarbox">SD to HD</button></span>
+  <div id="SD_HD_2" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Transform SD into HD with pillarbox</h3>
+          <p>Transform a SD video file with 4:3 aspect ratio into an HD video file with 16:9 aspect ration by correct pillarboxing.</p>
```
Comment (kieranjol) [inline index.html]: ratio, not ration

File: index.html [hunk 2]
```diff
@@ -205,7 +205,36 @@
...
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-filter:v "colormatrix=bt601:bt709, scale=1440:1080:flags=lanczos, pad=1920:1080:240:0"</dt><dd>set colour matrix, video scaling and padding<br/>Three filters are applied:
+              <ol>
+                <li>The luma coefficients are modified from SD video (according to Rec. 601) to HD video (according to Rec. 709) by a colour matrix. Note that today Rec. 709 is often used also for SD and therefore you may cancel this parameter.</li>
+                <li>The scaling filter (<code>scale=1440:1080</code>) works for both upscaling and downscaling. We use the Lanczos scaling algorithm (<code>flags=lanczos</code>), which is slower but gives better results than the default bilinear algorithm.</li>
+                <li>The padding filter (<code>pad=1920:1080:240:0</code>) completes the transformation from SD to HD.</li>
+              </ol></dd>
+            <dt>-c:a copy</dt><dd>re-encodes using the same audio codec<br/>
+            For silent videos you can replace <code>-c:a copy</code> by <code>-an</code>.</dd>
```
Comment (kieranjol) [inline index.html]: maybe 'replace _with_' instead of 'replace _by_'?

**Review by kieranjol — COMMENTED** [review]
Looks good to me Reto apart form two little things.

File: index.html (hunk 1 above)
Comment (retokromer) [reply index.html]: Thanks!

File: index.html (hunk 2 above)
Comment (retokromer) [reply index.html]: Thanks!

**Pull request merged**